src/ipc/handler.c

```diff
@@ -872,7 +886,7 @@ static int ipc_comp_value(uint32_t header, uint32_t cmd)
 		trace_ipc_error("ipc: comp %d not found", data.comp_id);
 		return -ENODEV;
 	}
-	
+
```

Comment:
?

Reply:
I might've accidentally run trailing-space removal, didn't notice.
I can remove that change if needed :)